protostar/starknet/cheatable_syscall_handler.py

```diff
@@ -202,36 +179,28 @@ def _deploy(
             constructor_calldata=constructor_calldata,
             deployer_address=deployer_address,
         )
-        # BEGIN: PROTOSTAR_MODIFICATION — update mappings
+
+        # --- MODIFICATIONS START --- # TODO
```

Comment:
What's the ponit of `TODO` without a description of what needs to be done?

Reply:
That's the pattern that was used everywhere else in codebase, I have simply unified things here.

Reply:
Replaced this with [`#region syntax`](https://stackoverflow.com/a/20225477)